src/main/java/frc/robot/subsystems/AmpBar.java

```diff
@@ -1,84 +1,93 @@
 package frc.robot.subsystems;
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 import com.ctre.phoenix.motorcontrol.NeutralMode;
-
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.RelativeEncoder;
 import frc.robot.Constants;
 import frc.robot.Robot;
+import swervelib.motors.SparkMaxSwerve;
 
 public class AmpBar {
-    enum AmpBarStates {
+    public enum AmpBarStates {
         IN,
-        OUT
+        SHOOTING,
+        OUT,
```

Comment:
Is this state ever used?

Reply:
Hopefully, it's used for the climber going up possibly I think?